src/com/javabasic/lesson14oop/CommercialRealEstateTax.java

```diff
@@ -0,0 +1,25 @@
+package com.javabasic.lesson14oop;
+
+public class CommercialRealEstateTax extends RealEstateTax {
+
+    double revenue;
+    private static final double maxRevenueBeforeTax = 100;
```

Comment:
constants should be highlighted in code with style, MAX_REVENUE_BEFORE_TAX

Reply:
Done